### ISSUE TITLE
Charter review process

### ISSUE THREAD

**Issue by mnot** [w3c/process#580]
Right now the Process is pretty lax about [how charters are initiated](https://www.w3.org/Consortium/Process/Drafts/#WGCharterDevelopment). Conceivably, a charter could be worked on for a week or two and then go to the AC without anyone outside a limited subset of the community being any wiser.

While in some aspects that's great, if we want broad review of charters to assure that there's strong consensus on them and that they align with values, we should make sure that they get broad review, and people who might have feedback have adequate chance to do so.

Some proposals:

- [ ] A new term, _charter under consideration_ be introduced to the process. While a charter can be under development privately, once it's _under consideration_ it is acknowledged that the W3C is considering adoption of the charter and commencing discussion and evaluation of it.
- [ ] Charters under consideration are announced to the entire community (e.g., using an e-mail list + RSS feed + web page), not just the AC.
- [ ] A community-wide venue be established for discussion of charters under consideration. Probably a GitHub repo, maybe a mailing list. Needs to be a community space, not a Team-oriented tracking mechanism like the strategy funnel.
- [ ] There is a minimum limit on the time a charter is under consideration baked into the process, to allow ample notice and time for discussion inside the W3C as well as with any external stakeholders. I'd suggest four weeks.
- [ ] Horizontal review be required by the Process, not just the Guide, and it be performed by designated CGs or IGs as appropriate, not just delegated to Team members.

**Comment by dwsinger** [comment]
I think having the Charters Under Consideration/Development "obviously visible" to the community, and especially the AC and the horizontal review groups and AB makes a lot of sense. I am not sure I want to add mandatory workload (e.g. getting HR signoffs); I feel we should try and see what visibility does for us, and allowing people to comment before it goes to the AC. The AC (and everyone, including team) would then be able to see "hang on, the i18n group has a major concern" and we could handle it. 

This sounds eminently do-able using modern tooling.

**Comment by nigelmegitt** [comment]
If it encourages AC engagement in Charter development, it is probably a Good Thing.

**Comment by OR13** [comment]
@cwilso 

> In short, we DO have this today - it's the charter review that the AC gets. 

My point was not to burden the AC with more work, it was to give them more than 1 chance to do that work well.

>  just calling out that you have to minimize the time required from all reviewers, 

Yes, part of that might involve spreading reviews time out, or giving more than 1 chance... 

Imagine 3 opportunities to review the charter instead of 1.... 
If a wg asks 3 times for a review, and the reviewer still don't have time... they have still done the same amount of work as if they are asked once, but the chances of them having a really bad week during all 3 asks is less.... 

Though some folks might not ever have enough time to review... at least giving them a few chances is better than 1 chance... especially if their review might prevent a WG from getting started with deliverables they could never do anything but object too.... WGs should not be capable of operating without known such objections are coming... since AC reviews seem like the only way to mitigate that, asking for more AC reviews up front (before charter / after charter) seems like the only defense against this...

Is there a less expensive solution?

**Comment by torgo** [comment]
There's been sone discussion on AC-Forum on what role the TAG should or shouldn't play regarding charter development and review. Shall we fold that in here as well?

**Comment by torgo** [comment]
Actually I see this was already discussed in https://github.com/w3c/w3process/issues/328 ?

**Comment by frivoal** [comment]
A few more considerations:
* Once a charter-under-consideration moves to being proposed to the AC, the communication announcing it to the AC (mail, wbs poll intro…) needs to point to a disposition of comments of some sort. I don't strongly care about whether it's a manually crafted document or a properly labelled set of github issues, but it must be possible for any AC rep casting their ballot to observe:
  * that broad discussion and review have happened
  * that issues raised were [addressed](https://www.w3.org/2021/Process-20211102/#formal-address)
  * whether the way issues were addressed were to the satisfaction of the people who raised them
* We should formally identify who's chairing this under-consideration phase. Particularly, it's important to tell the difference between those who are expected to express subjective opinions about should or shouldn't be written in the charter from those who're trying to asses whether we have consensus. Closing an issue against the wishes of the person who raised it may still happen, but it's very different whether that happens when a chair has determined that this was the only way to make progress while noting that doing so might cause a formal objection to be raised, versus when it's a proponent of the thing being proposed who wants to close because they like it the way it is.

Assuming we ensure the above, it may well be that the current set up of discussing charters in https://github.com/w3c/charter-drafts/ (and announcing that we do so) could largely fulfill the intent of @mnot 's proposal in the OC. Or maybe we do need more formality.

Either way, I do feel pretty strongly that we should make it trivially observable by the AC whether a charter that is being proposed is the result of extensive (and successful) consensus building, of bitter and unresolved discussions, or something few people have looked at.

**Comment by frivoal** [comment]
I'd consider it fuzzily related to director-free. We could improve this while still having a director, and we could switch away from having a director without fixing this. But it's a thing that the Director used to do that would be good to improve into something that no longer involves him, so yeah, it's mostly part of director-free. As for landing it in process-2022, I'd say we can, if we manage to find consensus on exactly what it is we want to do.

I see no reason to tie it to other parts of director-free, so even if the rest were not to be ready, if we agree on what to do here, there's no reason to refrain from landing it.

**Comment by dwsinger** [comment]
I don't see a need to tie either ProtoCharter visibility or better sign-off visibility (e.g. HR groups, TAG, AB) to Director-Free.

**Comment by plehegar** [comment]
Discussed on [June 8](https://www.w3.org/2022/06/08-w3process-minutes.html).

Suggested approach to move forward was a 2 steps:

1. replace the Director by the Team (FOs would go to the Council in any case)
2. look into fixing the pre-AC review period to avoid surprising the AC when starting a review. (no proposal yet).

Step 1 is now waiting on @frivoal  to create a new pull request. That pull request should take into account [@dwsinger 's comment](https://github.com/w3c/w3process/pull/586#issuecomment-1150063659) at the minimum.

We'll come back to step 2 after that (in the fall most probably).

**Comment by plehegar** [comment]
Discussion with @frivoal on how to address the pre-AC review period for charters:

3 phases of chartering:
    phase 1 - prior to Advance Notice
    phase 2 - between Advance Notice and AC Review
    phase 3 - post AC review (already documented in Process)


* Members have a right to ask the Team to start chartering (prior to phase 1), can Formally Object if the Team Decision disagree or the Team fails to answer within 3 months
* phase 2 starts when the Team sends the Advance Notice.
* The Chair of phase 2 MUST be identified by the Team in the advance notice
* Phase 2 is public, and anyone can send comments
* During phase 2, the Chair:
* MUST lead wide review (including HR)
* MUST formally address comments
* MUST produce a DoC
* Chair may proceed to AC review even if there is no consensus (dissent will be identified in the DoC)
* Default duration of chartering is 6 months. After 6 months, the chair must either:
  * Send the charter to the Team, requesting an AC Review
  * decide to extend the chartering (for a max of 6? months [at a time|combined])
  * decide to abandon the chartering if they see no reasonable path forward. (A lack of decision is equivalent to decision to abandon)
* Decision to extend or abandon must be announced to the AC, and may be FOed.
* The Team may deny an AC Review request by the Chair if the proposed charter does not fulfill the Process's requirements about Charters, or if the Chair did not fullfill the requirements of phase 2. That is a Team decision, which must be announced, and can be Formally Objected to.
* Any FO raised during phase 2 is processed together with FOs raised during the AC review (i.e. they do not block phase 2), except FOs against the decision to abandon or extend chartering, which are processed according to the usual deadlines (See Process 5.6.1.)

**Comment by plehegar** [comment]
Question: if a Council agrees with an FO against decision to abandon, who decides whether that means we must continue working on the charter, or if it means we must send the charter to AC Review as it is? Council? Team (=TILT)?
The Council itself deciding that would be unusual: Councils typically only decide uphold / override, not what to do instead. But putting that decision elsewhere would be a weird, though TILT could handle that (but then if you get an FO against TILT's decision, you get a second council in a row…)
The same question applies to an AC appeal against the Council's decision.
Similarly, if the Council (or an AC Appeal) agrees with an FO against decision to extend, who decides if we must abandon or if must move to AC?

**Comment by plehegar** [comment]
we should probably have a "last call" period before the end of  phase 2.

**Comment by mnot** [comment]
When you say 'Default duration of chartering' I think you mean 'Default duration of phase 2' -- correct?

'Chair' is confusing -- 'facilitator'? 'chartering lead'?

Other than that, this looks like a promising start. I'd like to see more about community engagement and discussion -- so many of our interactions are one-way.

**Comment by cwilso** [comment]
(as @plehegar and I were talking about this when he wrote it) yes, default duration of phase two, and I had the same feedback as you - that "chair" was confusing in context.  "Chartering facilitator" seems good.

**Comment by frivoal** [comment]
> you mean 'Default duration of phase 2' -- correct?

+1

> 'Chair' is confusing -- 'facilitator'? 'chartering lead'?

Well, I'm not too attached to what we call the person, but my idea is that we need to be clear about who is doing what, and not confuse the person(s) who are championing an idea, with the one who's trying to make sure that we address feedback properly and try to get to consensus where we can.

A "lead proponent" of a charter might want to close a change request with "yeah, no, that's not what we're trying to do here", while a chair-like figure would try to find the compromise that create the weakest objections. They might decide to go ahead without consensus if it seems unreachable, but they should strive for it, and not try to drive through a particular take that they personally favor.

To me, "chair" works for that. It's also convenient in a Process sense, as things like "Chairs decision" are already defined. If that doesn't work, I think "facilitator" could be ok too, but "lead" is too easily confused with the main person who is trying to make a particular take on the idea move forward. Charters obviously need proponents—if nobody cares it's not going to move forward—but I think distinguishing that from the person who's trying to make sure that the chartering activity is happening as it should is important.

**Comment by frivoal** [comment]
> we should probably have a "last call" period before the end of phase 2.

Yes, but just like for specs, it doesn't need to be formally encoded in the Process. It can be left as a good practice: when you're wrapping up, say so and call for a last round of feedback, otherwise you might miss important comments an only get them as FOs later.

**Comment by frivoal** [comment]
Landed PR https://github.com/w3c/process/pull/851 into the Editor's draft to address this issue. Additional concerns not yet discussed should be filed as separate issues.

**Comment by elena136t** [comment]
[Terms of Service.pdf](https://github.com/user-attachments/files/17745201/Terms.of.Service.pdf)
